src/plays/keeper/Keeper.css

```diff
@@ -0,0 +1,108 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.play-details .play-details-body {
+  font-family: "Montserrat", sans-serif;
```

Comment:
Why to use another font? Let the font come from the framework itself.

Reply:
Custom font family removed.